config_hostinit.go

```diff
@@ -61,20 +62,30 @@ func (c *HostInitConfig) Set() error {
 }
 
 func (c *HostInitConfig) ValidateAndDefault() error {
-	if c.HostThrottle <= 0 {
+	catcher := grip.NewSimpleCatcher()
+	if c.HostThrottle == 0 {
 		c.HostThrottle = defaultHostThrottle
+	} else if c.HostThrottle < 0 {
+		catcher.Add(errors.New("HostThrottle cannot be negative"))
```

Comment:
This statement can be simplified down to `catcher.New("HostThrottle cannot be negative")`.

nit: I would just simplify the entire else-if block down to `catcher.NewWhen(c.HostThrottle < 0, "HostThrottle cannot be negative")`.

Same thing with the other negative value checks.